File: notes/17.feb/typescript/FooTest.ts
import { Actions } from "./Action";

export class FooTest {

  test = "2";

  public testFoo = (): void => {
    const foo = 'Foo';
    const result = foo.toUpperCase();
    Actions.assertEquals(result + ' is not Foo', result, 'Foo');
    Actions.assertEquals('length should not differ', result.length, 4);
  }

  public foo = (): void => {
    const foo = 'Foo';
    const result = foo.toUpperCase();
    Actions.assertEquals(result + ' is not Foo', result, 'Foo');
    Actions.assertEquals('length should not differ', result.length, 4);
  }

  public testBar = (): void => { }

  public testFail = (): void => {
    Actions.assertEquals('asd', func(2), 10);
  }
}

const func = (n: number) => {
  return n
}

File: class exercises/17.feb/java/test/src/main/java/test/FooTest.java
package test.src.main.java.test;

import test.src.main.java.test.Action;

public class FooTest {

  public void testFoo() {
    String foo = "Foo";
    String result = foo.toUpperCase();
    Action.assertEquals("Result should be upper case", result, "FOO");
    Action.assertEquals("Length should not differ", result.length(), foo.length());
  }

  public void testBar() {
    System.out.println("Bar");
  }
}